cg/cli/demultiplex/finish.py

```diff
@@ -19,32 +19,25 @@ def finish_group():
     """Finish up after demultiplexing."""
 
 
-@finish_group.command(name="all")
+@finish_group.command(name="all-old-flow")
 @DRY_RUN
 @click.pass_obj
-def finish_all_cmd(context: CGConfig, dry_run: bool) -> None:
+def finish_all_cmd_old_flow(context: CGConfig, dry_run: bool) -> None:
     """Command to post-process all demultiplexed flow cells."""
```

Comment:
Should w extend doc str to clarify what old means?

Reply:
I updated the docstring and mentions that it uses the old api.